core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -366,6 +369,16 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
                 protocol, routingCtx, routingResult, req, getSSLSession(channel),
                 proxiedAddresses, clientAddress);
 
+        if (config.defaultServerNameResponseHeader()) {
+            reqCtx.addAdditionalResponseHeader(HttpHeaderNames.SERVER, "armeria");
```

Comment:
Let add version and os name using so that the header will be `Armeria/0.94.0 (Linux)`.
We can use `Version` and `System.getProperty("os.name")`.

Reply:
This value never changes so let's make a `private static final String` field for this.

Reply:
We should do this only when a user does not specify this. Let's consider that a user specifies a header in the same manner:
```java
reqCtx.addAdditionalResponseHeader(HttpHeaderNames.SERVER, "myServer");
```
Then the response header will contain both values which he/she does not expect.
We should add it [here](https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java#L202).
Please note how we add `HttpHeaderNames.CONTENT_LENGTH`. We should do it in the same way.

Reply:
question : 
now `Server` header like this
```
server: Armeria/0.94.1-SNAPSHOT (Linux)
```
Is it okay? I'm not sure `-SNAPSHOT` should be included.

Reply:
I think it doesn't matter because we always release without `-SNAPSHOT`. You can only see it while developing `Armeria` 😎

Reply:
I see, thank you! :grinning: